### ISSUE TITLE
Cannot parse more than one multi-line entry

### ISSUE THREAD

**Issue by wz4** [aredwood/zonefile-parser#39]
Unable to parse files that have more than one multi-line entry. Issue is in the [collapse_lines](https://github.com/aredwood/zonefile-parser/blob/master/zonefile_parser/helper.py#L143) function in helper.py

Recommendation:
Replace `buffer = ""` with `buffer = []` in code below.

```
def collapse_lines(lines:List[str],delimiter = ""):
    buffer = []
    collapsed_lines = []
    

    for line in lines:
        # if the single line has both a closing and 
        # opening bracket, then it can be added straight away
        # because it cannot be further collapsed
        if "(" in line and ")" in line:

            collapsed_lines.append(trim_brackets(line))

        # start of a multi-line record, store in buffer
        elif "(" in line:
            buffer.append(line)

        # add the line to the buffer, the buffer forms a single record
        # close the buffer
        elif ")" in line:
            buffer.append(line)

            # remove whitespace between quotes, between brackets
            collapsed_lines.append(delimiter.join(buffer))
            buffer = ""


        # if the buffer has content in it, add current line
        elif len(buffer) > 0:
            buffer.append(line)

        # record is not part of a multiline record, no alteration needed.
        else:
            collapsed_lines.append(line)
    return collapsed_lines
```

**Comment by aredwood** [comment]
have you got a sample file I can use for testing?

**Comment by wz4** [comment]
I believe this should be sufficient. 

```
@		IN	SOA	ns.example.com.	root.ns.example.com. ( 
					2024021612
					3600	; refresh (60 minutes)
					900	; retry (15 minutes)
					604800	; expire (7 days)
					900	; negative cache ( 15-min )
				)
				
; sample txt record will not parse
_dmarc.example.com.   3600   IN   TXT   (
        "v=DMARC1; p=none; sp=none; fo=1; rf=afrf;"
        "rua=mailto:test@example.com,mailto:test@example.com,mailto:test@example.com;"
        "ruf=mailto:test@example.com,mailto:test@example.com,mailto:test@example.com;")
        
sample	3600	IN	A	10.10.10.10	; sample A record
```

**Comment by wz4** [comment]
I discovered another issue I believe to be with the collapse_lines function. The SOA I provided does not parse correctly. The serial number and refresh interval get merged. I think the delimiter argument should be a space.

**Comment by aredwood** [comment]
released in 0.1.16 - tracking that new issue re merging of serial number / interval in #43
